rllib/models/preprocessors.py

```diff
@@ -77,11 +77,7 @@ def size(self):
     @property
     @PublicAPI
     def observation_space(self):
-        obs_space = gym.spaces.Box(
-            np.finfo(np.float32).min,
-            np.finfo(np.float32).max,
-            self.shape,
-            dtype=np.float32)
+        obs_space = gym.spaces.Box(-1., 1., self.shape, dtype=np.float32)
```

Comment:
Set to -1, 1 to be safer (it's a bit hard to compute the actual lo/hi correctly for complex obs spaces).

Reply:
Where is this observation space used? I think it might be better to have the observation space bounds to be `[-inf, inf]` in general because, as you mentioned, there are many observation spaces where there actually are no bounds. Do we sample from this observation space somewhere or why do things break with the old code?

Reply:
It's never used except for sampling from for initializing the loss.